src/shared/constants.js

```diff
@@ -1176,3 +1176,180 @@ export const hasGoalRatingTwoOrLess = (selectedGoals, crop = []) => {
 
   return crop.inactive || selectedGoals.every((rating) => crop[rating] <= 2);
 };
+
+export const getExperts = (councilId) => {
```

Comment:
rename to getExpertsData since sometimes it returns an array and sometimes it returns jsx

Reply:
or split it out into two functions, one for jsx and one for array of experts